content/index.html

```diff
@@ -32,13 +32,13 @@ <h2 class="subtitle no-anchor">The coupling library for partitioned multi-physic
     <div class="row">
       <div class="col-md-6">
         <p>
-          preCICE is an <strong>open-source coupling library</strong> for partitioned multi-physics simulations, including, but not restricted to fluid-structure interaction and conjugate heat transfer simulations.
+          preCICE is an <strong>open-source coupling library and ecosystem</strong> for general partitioned multi-physics and multi-scale simulations, including, surface coupling and volume coupling.
```

Comment:
```suggestion
          preCICE is an <strong>open-source coupling library and ecosystem</strong> for general partitioned multi-physics and multi-scale simulations, including surface coupling and volume coupling.
```

Reply:
I would even make it:


```suggestion
          preCICE is an <strong>open-source coupling library and ecosystem</strong> for general partitioned multi-physics and multi-scale simulations, including surface and volume coupling.
```

(to me, this reads more easily and compactly, even if more vaguely)